README.md

```diff
@@ -120,6 +120,13 @@ To run tests locally, you need to have [kind](https://kind.sigs.k8s.io/) install
 
 The command to execute the tests is: `VERSION=local make clean container kind-create-cluster test`.
 
+If building from mac, you need to split the command into two parts:
```

Comment:
pleas introduce a `## Building` paragraph.

Reply:
Sorry, I don't mean building here. There is no differences in building for this PR. I meant to add a paragraph for testing e2e. I will change the wording.